### PR TITLE
feat:add sessions route to admin

### LINKED ISSUE #160: Add admin/sessions page (UI only)
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

image
located at `admin/sessions`

Please go through the details of #153 before starting to work on this.

### PR DESCRIPTION
**Pull request by utkarshnath**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

screen shot 2017-06-10 at 6 13 22 pm

Fixes #160 
https://young-wildwood-91762.herokuapp.com/admin/sessions

### CODE REVIEW

File: app/routes/admin/sessions.js [hunk 1]
```diff
@@ -5,5 +5,47 @@ const { Route } = Ember;
 export default Route.extend({
   titleToken() {
     return this.i18n.t('Sessions');
+  },
+  model() {
+    return [{
+      event            : 'Event1',
+      title            : 'Title1',
+      speaker          : ['speaker1', 'speaker2'],
```
Comment (niranjan94) [inline app/routes/admin/sessions.js]: to be called `speakers`

File: app/routes/admin/sessions.js [hunk 2]
```diff
@@ -5,5 +5,47 @@ const { Route } = Ember;
 export default Route.extend({
   titleToken() {
     return this.i18n.t('Sessions');
+  },
+  model() {
+    return [{
+      event            : 'Event1',
```
Comment (niranjan94) [inline app/routes/admin/sessions.js]: Would be of the form, 

```js
event: {
   name: 'Event1'
}
```

File: app/routes/admin/sessions.js [hunk 3]
```diff
@@ -5,5 +5,47 @@ const { Route } = Ember;
...
+      title            : 'Title1',
+      speaker          : ['speaker1', 'speaker2'],
+      dateOfSubmissoin : new Date(),
```
Comment (niranjan94) [inline app/routes/admin/sessions.js]: would be `submittedAt`

File: app/routes/admin/sessions.js [hunk 4]
```diff
@@ -5,5 +5,47 @@ const { Route } = Ember;
...
+      startAt          : new Date(),
+      endAt            : new Date(),
+    },
+    {
+      event            : 'Event3',
+      title            : 'Title3',
+      speaker          : ['speaker1', 'speaker2'],
+      dateOfSubmissoin : new Date(),
+      startAt          : new Date(),
```
Comment (niranjan94) [inline app/routes/admin/sessions.js]: `startsAt`

File: app/routes/admin/sessions.js [hunk 5]
```diff
@@ -5,5 +5,47 @@ const { Route } = Ember;
...
+      endAt            : new Date(),
```
Comment (niranjan94) [inline app/routes/admin/sessions.js]: `endsAt`

File: app/routes/admin/sessions.js (hunk 1 above)
Comment (niranjan94) [inline app/routes/admin/sessions.js]: And would be of the form,
```js
speakers: [
  {
    name: 'Speaker1'
  },
  {
    name: 'Speaker2'
  },
]
```

**Comment by utkarshnath** [comment]
@niranjan94 plz check

**Comment by niranjan94** [comment]
This also should be made using dynamic route segments like 
`/admin/sessions` - All
`/admin/sessions/pending` - Pending
`/admin/sessions/accepted` - Accepted
`/admin/sessions/rejected` - Rejected
`/admin/sessions/deleted` - Deleted

But I'll open a new issue for that in the next milestone. This looks good for now.

**Pull request merged**